### ISSUE TITLE
Better display of Pandas DataFrame with multiindex

### ISSUE THREAD

**Issue by scls19fr** [spyder-ide/spyder#2513]
Hello,

Here is some code to make a Pandas DataFrame with multiindex

```
import pandas as pd
import numpy as np

def mklbl(prefix,n):
    return ["%s%s" % (prefix,i)  for i in range(n)]

def mi_sample():
    miindex = pd.MultiIndex.from_product([mklbl('A',4),
                                       mklbl('B',2),
                                       mklbl('C',4),
                                       mklbl('D',2)])


    micolumns = pd.MultiIndex.from_tuples([('a','foo'),('a','bar'),
                                        ('b','foo'),('b','bah'),
                                        ('c','foo'),('c','bah'),
                                        ('d','foo'),('d','bah'),
                                        ('e','foo'),('e','bah')],
                                         names=['lvl0', 'lvl1'])

    dfmi = pd.DataFrame(np.arange(len(miindex)*len(micolumns)).reshape((len(miindex),len(micolumns))),
                 index=miindex,
                 columns=micolumns).sortlevel().sortlevel(axis=1)
    return(dfmi)
```

It will be nice to improve display of Pandas DataFrame with multiindex.

![capture d ecran 2015-07-02 a 22 43 30](https://cloud.githubusercontent.com/assets/109167/8487417/443d2e28-210c-11e5-9ff2-33c4eb5eef76.png)

Sorry for this other issue / enhancement request.
That's probably a hard one.

Kind regards

**Comment by ccordoba12** [comment]
I don't know how easy this could be to do.

@dhoegh, would you mind to take a look at it?

**Comment by Prikers** [comment]
This may be a duplicate of #4018 (to be resolved through #3873)

**Comment by goanpeca** [comment]
@Prikers correct! closing
